src/node_modules/logger.js

```diff
@@ -405,7 +401,7 @@ export default class Logger extends Writable {
 			};
 
 			this[$reconnection].on('connect', function (con) {
-				DEBUG.log("Connected");
+				this.debugLogger.log("Connected");
```

Comment:
`this` here does not refer to the logger object I am afraid so it will throw an error.